internal/action/command.go

```diff
@@ -641,6 +645,9 @@ func (h *BufPane) ShowKeyCmd(args []string) {
 		return
 	}
 
+	// Convert possible received string of raw escape sequence
+	args[0] = ConvertRawEscFromString2Byte(args[0])
```

Comment:
I would use it directly without intermediate assignment, e.g. `TryBindKey(convertRawEscStr2Byte(args[0]), args[1], true)`.